src/misc.h

```diff
@@ -46,7 +46,9 @@ class Misc : public BeerXMLElement
    enum Type {Spice, Fining, Water_Agent, Herb, Flavor, Other}; // NOTE: BeerXML expects "Water Agent", but we can't have white space in enums :-/.
    //! \brief Where the ingredient is used.
    enum Use { Boil, Mash, Primary, Secondary, Bottling };
-   Q_ENUMS( Type Use )
+   //! \brief What is the type of amount.
+   enum AmountType { Weight, Volume };
```

Comment:
I know this isn't done above, but please prefix these enumerations like `AmountType_Weight` and `AmountType_Volume`.